### PR TITLE
Update getting-started.md

### PR DESCRIPTION
**Pull request by yingbaiibm**
update default KUBEVIRT_PROVIDER version

**What this PR does / why we need it**:
Update doc info for getting-started.md. 
I followed the getting-started.md to setup development env and found that k8s-1.18 is no longer exist and the current default version of KUBEVIRT_PROVIDER is k8s-1.20. Need update the doc. 

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
None
```
Signed-off-by: Ying Bai yingbai@cn.ibm.com

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ dockerized environment, clone the KubeVirt repository, `cd` into it, and:
 
 ```bash
 # Build and deploy KubeVirt on Kubernetes in our vms inside containers
-export KUBEVIRT_PROVIDER=k8s-1.18 # this is also the default if no KUBEVIRT_PROVIDER is set
+export KUBEVIRT_PROVIDER=k8s-1.20 # this is also the default if no KUBEVIRT_PROVIDER is set
```
Comment (rmohr) [inline docs/getting-started.md:76]: @yingbaiibm how about removing this line? we will choose the current default anyway. This way we avoid that this line gets outdated.

Comment (yingbaiibm) [reply docs/getting-started.md:76]: That's also a choice.   But how about changing the comment to  "Find supported K8s version under KUBEVIRT_TOP/cluster-up/cluster and set required KUBEVIRT_PROVIDER.  If no KUBEVIRT_PROVIDER is set, default value will be used"      In this way, user will be clear about how to set the KUBEVIRT_PROVIDER and choose the version they want.

Comment (dankenigsberg) [inline docs/getting-started.md:76]: How about a shorter
```suggestion
# export KUBEVIRT_PROVIDER=k8s-1.20 #  uncomment to use a non-default KUBEVIRT_PROVIDER
```

Comment (rmohr) [reply docs/getting-started.md:76]: Works for me too. Anything which does not mention an explicit version (and therefore needing a regular bump) is an improvement.

**Pull request closed without merging**